src/content/en/updates/posts/2015/11/app-shell.markdown

```diff
@@ -127,77 +136,79 @@ During the first load experience, your goal is to get meaningful content to the
 
 # A Practical Implementation
 
-We’ve written a fully working [sample](https://github.com/GoogleChrome/application-shell) using the Application Shell architecture using vanilla ES2015 JavaScript for the client and Express.js for the server. There is course nothing stopping you from using your own stack for either the client or the server portions (e.g PHP, Ruby, Python). We found using JS all the way through relatively painless for this example.
+We’ve written a fully working [sample](https://github.com/GoogleChrome/application-shell) using the application shell architecture, vanilla ES2015 JavaScript for the client, and Express.js for the server. There is of course nothing stopping you from using your own stack for either the client or the server portions (e.g PHP, Ruby, Python). We found using JS all the way through relatively painless for this example.
```

Comment:
I'd drop "We found using JS all the way through relatively painless for this example."